### PR TITLE
[BEL-2023] Update sponsors and add link to registration

### PR DESCRIPTION
**Pull request by pigor**

### CODE REVIEW

File: content/events/2019-belem/registration.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-+++
+**+++
```
Comment (adrianmoisey) [inline content/events/2019-belem/registration.md:1]: Is this change to 2019 intended?

Comment (phrawzty) [reply content/events/2019-belem/registration.md:1]: @adrianmoisey This just breaks things. Fairly certain it's an error.

Comment (phrawzty) [reply content/events/2019-belem/registration.md:1]: @adrianmoisey This just breaks things. Fairly certain it's an error.

image

File: content/events/2019-belem/registration.md [hunk 2]
```diff
@@ -9,3 +9,4 @@ icons = "true"
 ### <b>Inscreva-se</b>
 
 [Link para Inscrição](https://www.sympla.com.br/devopsdays-belem-2019__666901)
+**
```
Comment (phrawzty) [inline content/events/2019-belem/registration.md:12]: Continuation of error from line 1 above…

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: content/events/2019-belem/registration.md (hunk 1 above)
Comment (pigor) [reply content/events/2019-belem/registration.md]: My bad, it was a ctrl + b in a wrong place. PR updated!

File: content/events/2019-belem/registration.md (hunk 2 above)
Comment (pigor) [reply content/events/2019-belem/registration.md]: My bad, it was a ctrl + b in a wrong place. PR updated!

**Pull request merged**